<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnAfim">
      <property name="text">
       <string>Afim</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnQuad">
      <property name="text">
       <string>Quadrática</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Exponencial</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnLog">
      <property name="text">
       <string>Logaritma</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEscolha_a_fun_o_desejada">
    <property name="title">
     <string>Escolha a função desejada</string>
    </property>
   </widget>
   <addaction name="menuEscolha_a_fun_o_desejada"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
